### ISSUE TITLE
Travis error with OS X recipes for R; dyld: Library not loaded: @rpath/libintl.9.dylib

### ISSUE THREAD

**Issue by sebastian-luna-valero** [conda-forge/staged-recipes#3452]
Hello,

On [this](https://travis-ci.org/conda-forge/staged-recipes/builds/262190705#L440) travis build, I am getting:
```
dyld: Library not loaded: @rpath/libintl.9.dylib
  Reason: image not found
```

I have tried including the following line in the `build.sh` file:
```
export LDFLAGS="${LDFLAGS} -Wl,-rpath,$PREFIX/lib"
```

as suggested [here](https://github.com/conda-forge/staged-recipes/pull/1445#discussion_r83709368) but it doesn't work and I don't know what else to try.

Could you please let me know your thoughts?

Best regards,
Sebastian

**Comment by isuruf** [comment]
`defaults` channel has added `r-base-3.4.1` and therefore conda-smithy is trying to create a matrix for `CONDA_R=3.4.1` and `CONDA_R=3.3.2`, but `r-base-3.4.1` is not compatible with the `gettext` version in conda-forge. 2 options, build `r-base-3.4.1` or patch `conda-smithy` to not consider `r-base-3.4.1` in the matrix of R versions.

cc @conda-forge/core @bgruening @johanneskoester

**Comment by bgruening** [comment]
We should get a r-base-3.4.1 into conda-forge. As I remember our last conversation, staying with the r-base-3.X.1 release is the way to go.

**Comment by nsoranzo** [comment]
Can this be solved with high priority? It's blocking a few R recipes.

Maybe temporarily modify `scripts/run_docker_build.sh` as below:
```
diff --git a/scripts/run_docker_build.sh b/scripts/run_docker_build.sh
index d9cf7a3b..34286798 100755
--- a/scripts/run_docker_build.sh
+++ b/scripts/run_docker_build.sh
@@ -64,5 +64,5 @@ find ~/conda-recipes -mindepth 2 -maxdepth 2 -type f -name "yum_requirements.txt
     | xargs -n1 cat | grep -v -e "^#" -e "^$" | \
     xargs -r /usr/bin/sudo -n yum install -y
 
-conda build-all ~/conda-recipes --matrix-conditions "numpy >=1.11" "python >=2.7,<3|>=3.5" "r-base >=3.3.2"
+conda build-all ~/conda-recipes --matrix-conditions "numpy >=1.11" "python >=2.7,<3|>=3.5" "r-base ==3.3.2"
 EOF
```

while the new r-base-3.4.1 is finalised?

**Comment by johanneskoester** [comment]
Although I have done the r-base migration before, I won't be able to do it this time, at least not until later in September (I have just started a new position and need to organize a lot of things). In principle, all one has to do is copy the r-base recipe from conda/conda-recipes, and modify the pinnings so that they fit to what we have in conda-forge. Would anybody volunteer to do this? I am happy to review PRs.

**Comment by isuruf** [comment]
@nsoranzo, that patch looks okay. What happens when creating a feedstock? Does it get a R matrix?

**Comment by nsoranzo** [comment]
@isuruf I haven't tried it :blush: 
You may hijack https://github.com/conda-forge/staged-recipes/pull/3437 if you want, or I can ask for push permission for the branch to my colleague @wjurkowski.

**Comment by bgruening** [comment]
Aspiring PR is here: https://github.com/conda-forge/r-base-feedstock/pull/21

**Comment by nsoranzo** [comment]
Close?

**Comment by bgruening** [comment]
@sebastian-luna-valero can we close this one or do you still have problem?

**Comment by sebastian-luna-valero** [comment]
Thanks all for your input. I am on holiday until the end of August, can I have a look when I am back, please?

**Comment by jakirkham** [comment]
Should this be moved to the `r-base` feedstock?

**Comment by sebastian-luna-valero** [comment]
Many thanks all for your great work. Closing issue now.

**Comment by sebastian-luna-valero** [comment]
I spoke too soon. 4 out of my 5 recipes are now passing the travis tests but one of them still fails apparently due to the same problem. Or I am not able to find any other reason. 

The package is `betareg`:
https://cran.r-project.org/package=betareg

It fails when building it against R 3.4.1

Could somebody please have a look at the travis output [here](https://travis-ci.org/conda-forge/staged-recipes/builds/262188074)?

**Comment by isuruf** [comment]
This is a duplicate of https://github.com/conda-forge/conda-forge.github.io/issues/434
See point 2 there.
